client/components/faq/docs/example.jsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FAQ from 'components/faq';
+import i18n from 'lib/mixins/i18n';
+
+const faqItems = [
+	{
+		question: 'Do you sell domains?',
+		answer: i18n.translate(
```

Comment:
You're translating the answer, but not the question. I actually wouldn't translate either for this example, because once this gets committed and pushed to master, these strings will be submitted for translation for real. I don't think that should happen until we're ready to implement the final wording in the component that will be displayed to users.